plugins/woocommerce-admin/client/payments-welcome/style.scss

```diff
@@ -165,6 +178,72 @@
 				border-radius: 3px;
 			}
 		}
+
+		.connect-account__apms {
+			.components-card__header {
+				padding: $gap-large 10px $gap $gap-large;
```

Comment:
_Nitpick_: This padding should be `$gap-small $gap-large` to match the design. 📐 🙂

Reply:
Good catch! Fixed in https://github.com/woocommerce/woocommerce/pull/34581/commits/76a0313119e8c9c7b5f34d8c90a85448fe5e08ba.